src/components/ListItem.jsx

```diff
@@ -2,9 +2,28 @@ import { useState } from 'react';
 import './ListItem.css';
 import { updateItem, deleteItem } from '../api';
 import { calculateDateNextPurchased, ONE_DAY_IN_MILLISECONDS } from '../utils';
+import {
+	Restore as OverdueIcon,
+	RestartAlt as SoonIcon,
+	RadioButtonUnchecked as KindOfSoonIcon,
+	RemoveCircle as NotSoonIcon,
+	RadioButtonChecked as InactiveIcon,
+} from '@mui/icons-material';
 
 const currentDate = new Date();
```

Comment:
Curious why this is outside the ListItem component?

Reply:
great question! I have been trying to move variables outside of components to avoid their reinitialization on every re-render, but I will move it back in if necessary, let me know!